sources/zarr/large_image_source_zarr/__init__.py

```diff
@@ -52,12 +56,19 @@ def __init__(self, path, **kwargs):
         """
         super().__init__(path, **kwargs)
 
+        if str(path).startswith(NEW_IMAGE_PATH_FLAG):
+            self._initNew(path, **kwargs)
+        else:
+            self._initOpen(**kwargs)
+        self._tileLock = threading.RLock()
+
+    def _initOpen(self, **kwargs):
         self._largeImagePath = str(self._getLargeImagePath())
         self._zarr = None
         if not os.path.isfile(self._largeImagePath) and '//:' not in self._largeImagePath:
             raise TileSourceFileNotFoundError(self._largeImagePath) from None
         try:
-            self._zarr = zarr.open(zarr.SQLiteStore(self._largeImagePath), mode='r')
+            self._zarr = zarr.open(zarr.DirectoryStore(self._largeImagePath), mode='r')
```

Comment:
I think we need to leave this line as trying to open an SQLiteStore.  `zarr.open(path)` will work with Directory and Zip stores (and being pointed to the zarr files (.zattrs, at least) within a directory), but won't open an sqlite store.

Reply:
Changed back in [5317daa](https://github.com/girder/large_image/pull/1446/commits/5317daaa3ab2d8b89734fe14470df3f5d9902c1d)